<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GammaRay::FontBrowserWidget</class>
 <widget class="QWidget" name="GammaRay::FontBrowserWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>742</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_15">
   <item>
    <widget class="QSplitter" name="splitter_10">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QTreeView" name="fontTree">
      <property name="uniformRowHeights">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget_7">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLineEdit" name="fontText">
         <property name="text">
          <string>The quick brown fox jumps over the lazy dog</string>
         </property>
         <property name="placeholderText">
          <string>Enter some text here...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="fontOptions" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Point Size:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="pointSize"/>
          </item>
          <item>
           <widget class="QCheckBox" name="boldBox">
            <property name="text">
             <string>Bold</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="italicBox">
            <property name="text">
             <string>Italic</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="underlineBox">
            <property name="text">
             <string>Underline</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QTreeView" name="selectedFontsView"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
